express-basic/server.js

```diff
@@ -34,8 +30,11 @@ app.use("/color/:name", function (req, res) {
     case "blue":
       code = "#0000ff";
       break;
-    default:
+    case "ForestGreen":
+      code = "#228b22";
       break;
+    default:
+      console.log(`Sorry, we are out of ${name} color.`);
```

Comment:
this will not be returned to the browser, just printed on the console

Reply:
Ok!